### PR TITLE
im done

### PR DESCRIPTION
**Pull request by oiskritsky**
Done. but not all, haven't power in the end

### CODE REVIEW

File: js/main.js [hunk 1]
```diff
@@ -23,8 +23,8 @@
 // test = Boolean(test);
 // console.log(test); // True
 
-let arr = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20];
-let result = 0;
+// let arr = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20];
```
Comment (ghost) [inline js/main.js]: use `const`

File: js/main.js [hunk 2]
```diff
@@ -42,8 +42,80 @@ let result = 0;
...
+// };
+
+// const max = (a, x) => {
+// 	if (a < x) {
+// 		maxResult = x;
+// 		return maxResult;
+// 	} else if (a > x) {
+// 		maxResult = a;
+// 		return maxResult;
+// 	} else a == x;
```
Comment (ghost) [inline js/main.js]: `else a == x`  is redundant condition 
`else ` - more clear

Comment (ghost) [reply js/main.js]: and use `===` insted `==`

File: js/main.js [hunk 3]
```diff
@@ -42,8 +42,80 @@ let result = 0;
...
+
+///////////////////////////////////////
+///////////REPLACEMENT/////////////////
+///////////////////////////////////////
+
+const arr = [1, 17, 20, 31, 40, 68, 99, 100, 6, 10];
+const replaceTo = 'zero';
+
+const replaceNulltoZero = (array, word) => {
+	array.sort((a, b) => a - b);
```
Comment (ghost) [inline js/main.js]: Seems to sort arr isn't necessary.

File: js/main.js (hunk 1 above)
Comment (oiskritsky) [reply js/main.js]: fix

File: js/main.js (hunk 2 above)
Comment (oiskritsky) [reply js/main.js]: fix

File: js/main.js (hunk 3 above)
Comment (oiskritsky) [reply js/main.js]: i used sorting for the convenience of viewing the contents of the array

Comment (oiskritsky) [reply js/main.js]: deleted sort

**Pull request merged**